res/css/views/rooms/_RoomHeader.pcss

```diff
@@ -56,7 +56,7 @@ limitations under the License.
     min-width: 0;
     margin: 0 20px 0 16px;
     padding-top: 6px;
-    border-bottom: 1px solid $system;
+    border-bottom: 1px solid var(--cpd-color-gray-500);
```

Comment:
This will break custom themes e.g. https://github.com/aaronraimist/element-themes